.github/workflows/trivy-scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trivy Scan
+
+on:
+  push:
+    # Run on main and supported release branches
+    branches:
+      - main
+      - release-1.23
+      - release-1.22
+      - release-1.21
+  # Run weekly
+  schedule:
+    - cron: '0 12 * * 1'
+  # Allow manual runs
+  workflow_dispatch:
+
+jobs:
+  trivy-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aquasecurity/trivy-action@v0.8.0
+        with:
+          scan-type: 'fs'
+          format: 'sarif'
+          output: 'trivy-results.sarif'
+          ignore-unfixed: true
+          severity: 'HIGH'
```

Comment:
should this be 'CRITICAL,HIGH'? (ref. https://github.com/aquasecurity/trivy-action#scan-ci-pipeline)

Reply:
ah yeah good catch, thought it was this level and higher, but should include all